template.yaml

```diff
@@ -1118,3 +1139,28 @@ Resources:
             SlackAlertLambdaArn: !GetAtt SlackAlertLambdaFunction.Arn
           }
       RoleArn: !GetAtt StatesExecutionRole.Arn
+
+  DBRestoreStateMachineEventRule:
+    Type: AWS::Events::Rule
+    Properties:
+      Description: Trigger the DBRestoreStateMachine
+      Name: !Sub ${ProjectName}-DBRestoreStateMachine-rule
+      ScheduleExpression: "cron(0 8 ? * FRI *)" # Runs every FRIDAY at 8:00 AM UTC
```

Comment:
@RiyaJohn quick question on this. the schedule expression is commonly set for all the targets? in the present set of changes, it is not possible to have a unique cron for every target rds instance right? is that something we can consider?

Reply:
Schedule expression can be set only once per event rule. If targets need to run at different CRONs then users have to add more event rules.